_rules/image-no-text-0va7u6.md

```diff
@@ -26,11 +26,12 @@ acknowledgments:
 This rule applies to any HTML element that is [visible][] for which one of the following is true:
 
 - the element is an `img` element where at least one of the [image sources][] in its [source set][] does not reference an SVG document; or
-- the element is an `input` element in the [Image Button][] state and its `src` [attribute value][] does not reference an SVG document.
+- the element is an `input` element with a `type` [attribute value][] of `image` and its `src` [attribute value][] does not reference an SVG document; or
+- the element has a [`background-image`][background-image] CSS property with at least one [`image`][css-image] that is a [url reference][url-reference] that does not reference an SVG document.
```

Comment:
Which value of the property do you consider? I guess the computed value (after resolving cascade and such).